elm.json

```diff
@@ -0,0 +1,18 @@
+{
+    "type": "package",
+    "name": "damienklinnert/elm-spinner",
+    "summary": "A highly configurable, efficiently rendered spinner component",
+    "license": "MIT",
+    "version": "3.0.1",
```

Comment:
As mentioned in the PR intro, this version should change since this package will no longer work with Elm 0.18.